07-html.Rmd

```diff
@@ -408,3 +408,60 @@ In Section \@ref(latex-hardcore), we mentioned that if you feel the constraint o
+
+- [**shinyapps.io**](https://www.shinyapps.io/) allows for hosting dynamic content which requires a server running R. For example, one can host interactive R Markdown documents which use the `runtime: shiny` option to include Shiny components. shinyapps.io is an analog to RPubs for Shiny applications. Apps and interactive R Markdowns can be published using the push-button in the RStudio IDE or the `rsconnect::deployApp()` function. See the user guide (https://docs.rstudio.com/shinyapps.io/index.html) for more details. 
+
+- [**bookdown.org**](https://bookdown.org/home/about/) offers free hosting specifically for books written with the **bookdown** package. You may easily publish static output files of your book using the `bookdown::publish_book()` function. This platform is based on RStudio PBC's enterprise RStudio Connect product (see below).
+
+- [**RStudio Connect**](https://rstudio.com/products/connect/) is an enterprise product which organizations may run on their own servers. It can host a wide variety of content created in R (R Markdown documents, Shiny apps, APIs) in a secured environment with document-level access controls, viewership history, and more. Similar to the open-source RPubs and shinyapps.io offerings, content can be published to RStudio Connect using manual upload, the **rsconnect** package, or with Git-based deployment. 
+
+### Static website services
+
+In addition to R-specific services, you may host your HTML document on many freely available static website hosting services. These include:
+
+- [**GitHub Pages**](https://pages.github.com/) makes is particularly easy to publish markdown and HTML content straight from a GitHub repository. You may specify whether to host content from either your main branch's root, your main branch's `docs/` folder, or a specific `gh-pages` branch. You can publish new content simply by pushing new HTML files to your repository.
+
+- [**GitLab Pages**](https://docs.gitlab.com/ee/user/project/pages/) offers similar functionality to GitHub Pages for GitLab repositories. 
+
+- [**Netlify**](https://www.netlify.com/) is a platform to build and deploy static website content. It is a popular choice for web content created by the **blogdown** and **pkgdown** packages, but it can host all types of markdown and HTML files. Compared to GitHub Pages, it offers more advanced features such as deploy previews and compatability with all major static site generators (e.g. Hugo). The deployment process is just as easy as GitHub Pages and has many different options including drag-and-drop, command line, or Git integration. See the Netlify documentation (https://docs.netlify.com/) or the RStudio webinar "Sharing on Short Notice" (https://rstudio.com/resources/webinars/sharing-on-short-notice-how-to-get-your-materials-online-with-r-markdown/) for more details. 
+
+### Advice
```

Comment:
Should we add also in this part a small paragraph about "Automate the publishing" using CICD-workflow ? 
Seems not easy to mention this without saying more though... 🤔

Reply:
I'm probably already using too much jargon here with words like "deploy previews" and "static site generators". Maybe I will instead try to mention benefits to workflow instead of the specific names of things themselves. 

What do you think if I replace this

> Compared to GitHub Pages, it offers more advanced features such as deploy previews and compatability with all major static site generators (e.g. Hugo). The deployment process is just as easy as GitHub Pages and has many different options including drag-and-drop, command line, or Git integration

with this

> There are many different publishing options including including drag-and-drop, command line, or automated publishing from GitHub and GitLab repositories. Additionally, Netlify offers many helpful features such as website previews in pull requests.

?

Reply:
yes the second one sounds good ! it covers a lot with less specific naming!